src/components/Ecuador.js

```diff
@@ -13,27 +13,21 @@ import {
 import Spinner from './Spinner';
 
 const Ecuador = ({ t }) => {
-  const { provinces, isLoading, error } = useStats();
+  const { provinces, lastDayTotals, isLoading, isError } = useStats();
```

Comment:
Remove isLoading and isError from this component.

Reply:
Done